integrated_channels/moodle/client.py

```diff
@@ -39,7 +40,7 @@ def inner(self, *args, **kwargs):
             # On course creation (and ONLY course creation) success,
             # Moodle returns a list of JSON objects, because of course it does.
             # Otherwise, it fails instantly and returns actual JSON.
-            return response
+            return body
```

Comment:
ohhhh this is why my responses changed. I didn't even remember doing this. I should double check grades' responses.

Reply:
oh yes looks like body is what we want here since it's the json